public/assets/js/book/wizard/author.js

```diff
@@ -4,14 +4,7 @@ var lastSetAuthorInfix;
 var formValidator;
 
 function validateForm() {
-    formSubmitting = true;
-    var errorMessage;
-    if (errorMessage) {
-        showError("error-div", errorMessage);
-        return false;
-    }
-    hideError("error-div");
-    return true;
+    return validate(function(){ return null; })
```

Comment:
No need to pass an empty function. Passing nothing will result in the same behaviour. Same for other files below